rebalance/rebalance.py

```diff
@@ -295,10 +304,20 @@ def rebalance(plugin, outgoing_scid, incoming_scid, msatoshi: Millisatoshi = Non
                 excludes.append(worst_channel['channel'] + '/' + str(worst_channel['direction']))
                 continue
 
-            rpc_result = {"sent": msatoshi + fees, "received": msatoshi, "fee": fees, "hops": len(route),
-                          "outgoing_scid": outgoing_scid, "incoming_scid": incoming_scid, "status": "complete",
-                          "message": f"{msatoshi + fees} sent over {len(route)} hops to rebalance {msatoshi}"}
-            plugin.log("Sending %s over %d hops to rebalance %s" % (msatoshi + fees, len(route), msatoshi), 'debug')
+
+            rpc_result = {
+                "sent": msatoshi + fees,
+                "received": msatoshi,
+                "fee": fees,
+                "hops": len(route),
+                "outgoing_scid": outgoing_scid,
+                "incoming_scid": incoming_scid,
+                "status": "complete",
+                "message": f"{msatoshi + fees} sent over {len(route)} hops to rebalance {msatoshi}",
+            }
+            midroute_str = reduce(lambda x,y: x + " -> " + y, map(lambda r: get_node_alias(r['id']), route_mid))
+            full_route_str = "%s -> %s -> %s -> %s" % (get_node_alias(my_node_id), get_node_alias(outgoing_node_id), midroute_str, get_node_alias(my_node_id))
+            plugin.log("%d hops and %s fees for %s along route: %s" % (len(route), fees.to_satoshi_str(), msatoshi.to_satoshi_str(), full_route_str))
```

Comment:
This is cool, but wouldn't it be better to also output it in the rpc_result, and not only in the clightning log file?